build/configs/artik053s/st_things/defconfig

```diff
@@ -0,0 +1,1229 @@
+CONFIG_S5J_SPI=y
+# CONFIG_S5J_WATCHDOG is not set
+CONFIG_S5J_SFLASH=y
+# CONFIG_S5J_SENSOR_PPD42NS is not set
+
+#
+# Architecture Options
+#
+# CONFIG_ARCH_NOINTC is not set
+# CONFIG_ARCH_VECNOTIRQ is not set
+# CONFIG_ARCH_DMA is not set
+# CONFIG_ARCH_HAVE_IRQPRIO is not set
+# CONFIG_ARCH_L2CACHE is not set
+# CONFIG_ARCH_HAVE_COHERENT_DCACHE is not set
+# CONFIG_ARCH_HAVE_ADDRENV is not set
+# CONFIG_ARCH_NEED_ADDRENV_MAPPING is not set
+CONFIG_ARCH_HAVE_VFORK=y
+# CONFIG_ARCH_HAVE_MMU is not set
+CONFIG_ARCH_HAVE_MPU=y
+# CONFIG_ARCH_NAND_HWECC is not set
+# CONFIG_ARCH_HAVE_EXTCLK is not set
+# CONFIG_ARCH_HAVE_POWEROFF is not set
+CONFIG_ARCH_HAVE_RESET=y
+CONFIG_ARCH_USE_MPU=y
+CONFIG_ARCH_STACKDUMP=y
+# CONFIG_ENDIAN_BIG is not set
+# CONFIG_ARCH_IDLE_CUSTOM is not set
+CONFIG_ARCH_CUSTOM_PMINIT=y
+# CONFIG_ARCH_HAVE_RAMFUNCS is not set
+# CONFIG_ARCH_HAVE_RAMVECTORS is not set
+
+#
+# Board Settings
+#
+CONFIG_BOARD_LOOPSPERMSEC=29100
+# CONFIG_ARCH_CALIBRATION is not set
+
+#
+# Interrupt options
+#
+CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
+CONFIG_ARCH_INTERRUPTSTACK=0
+# CONFIG_ARCH_HAVE_HIPRI_INTERRUPT is not set
+
+#
+# Boot options
+#
+# CONFIG_BOOT_RUNFROMEXTSRAM is not set
+CONFIG_BOOT_RUNFROMFLASH=y
+# CONFIG_BOOT_RUNFROMISRAM is not set
+# CONFIG_BOOT_RUNFROMSDRAM is not set
+# CONFIG_BOOT_COPYTORAM is not set
+
+#
+# Boot Memory Configuration
+#
+CONFIG_RAM_START=0x02023800
+CONFIG_RAM_SIZE=968704
+# CONFIG_ARCH_HAVE_SDRAM is not set
+
+#
+# Board Selection
+#
+# CONFIG_ARCH_BOARD_ARTIK053 is not set
+CONFIG_ARCH_BOARD_ARTIK053S=y
+# CONFIG_ARCH_BOARD_ARTIK055S is not set
+# CONFIG_ARCH_BOARD_SIDK_S5JT200 is not set
+CONFIG_ARCH_BOARD_ARTIK05X_FAMILY=y
+CONFIG_ARCH_BOARD="artik05x"
+
+#
+# Common Board Options
+#
+# CONFIG_BOARD_CRASHDUMP is not set
+CONFIG_LIB_BOARDCTL=y
+CONFIG_BOARDCTL_RESET=y
+# CONFIG_BOARDCTL_UNIQUEID is not set
+# CONFIG_BOARD_FOTA_SUPPORT is not set
+
+#
+# Board-Specific Options
+#
+CONFIG_ARTIK05X_BOOT_FAILURE_DETECTION=y
+CONFIG_ARTIK05X_BOOT_COUNTS_ADDR=0x80090810
+CONFIG_ARTIK05X_FLASH_CAPACITY=8388608
+CONFIG_ARTIK05X_FLASH_PAGE_SIZE=4096
+CONFIG_ARTIK05X_FLASH_PART=y
+CONFIG_ARTIK05X_FLASH_MINOR=0
+CONFIG_ARTIK05X_FLASH_PART_LIST="16,48,192,32,512,2400,1536,1536,1000,400,8,512,"
+CONFIG_ARTIK05X_FLASH_PART_TYPE="none,ftl,none,none,none,none,none,ftl,smartfs,romfs,config,none,"
+CONFIG_ARTIK05X_FLASH_PART_NAME="bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,rom,nvram,sssrw,"
+CONFIG_ARTIK05X_AUTOMOUNT=y
+CONFIG_ARTIK05X_AUTOMOUNT_USERFS=y
+CONFIG_ARTIK05X_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p8"
+CONFIG_ARTIK05X_AUTOMOUNT_USERFS_MOUNTPOINT="/mnt"
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW=y
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_DEVNAME="/dev/smart0p10"
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_MOUNTPOINT="/sss"
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS=y
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_DEVNAME="/dev/mtdblock9"
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_MOUNTPOINT="/rom"
+
+#
+# Kernel Features
+#
+# CONFIG_DISABLE_OS_API in not set
```

Comment:
Don't modify config manually. Update this file through menuconfig.

Reply:
I have reset it in menuconfig.